app/src/renderer/vuex/modules/transactions.js

```diff
@@ -48,19 +52,26 @@ export default ({ node }) => {
     },
     async getAllTxs({ commit, dispatch }) {
       commit(`setHistoryLoading`, true)
-      const stakingTxs = await dispatch(`getTx`, `staking`)
-      commit(`setStakingTxs`, stakingTxs)
 
-      const governanceTxs = await dispatch(`getTx`, `governance`)
-      commit(`setGovernanceTxs`, governanceTxs)
+      try {
+        const stakingTxs = await dispatch(`getTx`, `staking`)
+        commit(`setStakingTxs`, stakingTxs)
 
-      const walletTxs = await dispatch(`getTx`, `wallet`)
-      commit(`setWalletTxs`, walletTxs)
+        const governanceTxs = await dispatch(`getTx`, `governance`)
+        commit(`setGovernanceTxs`, governanceTxs)
 
-      const allTxs = stakingTxs.concat(governanceTxs.concat(walletTxs))
-      await dispatch(`enrichTransactions`, {
-        transactions: allTxs
-      })
+        const walletTxs = await dispatch(`getTx`, `wallet`)
+        commit(`setWalletTxs`, walletTxs)
+
+        const allTxs = stakingTxs.concat(governanceTxs.concat(walletTxs))
+        await dispatch(`enrichTransactions`, {
+          transactions: allTxs
+        })
+      } catch (error) {
```

Comment:
i think we need to be catching errors in all the vuex modules - @faboweb @NodeGuy @fedekunze

Reply:
We should make sure, that the resulting errors appear in some log and are sent via Raven. So that we can act on odd behavior with remote clients.